build.gradle

```diff
@@ -40,7 +40,7 @@ repositories {
 }
 
 // versions for the dependencies
-final gatkVersion = '4.alpha.2-53-g3e20270-SNAPSHOT'
+final gatkVersion = '4.alpha.2-82-gbc8318e-SNAPSHOT'
```

Comment:
Change for https://github.com/broadinstitute/gatk/commit/0e0108384b0f0d0d35d5d9ca2076b5d44de59add